packages/boxplot/src/BoxPlotTooltip.tsx

```diff
@@ -0,0 +1,74 @@
+import { BoxPlotSummaryFormatted, BoxPlotTooltipProps, BoxPlotSummary } from './types'
+import { Chip } from '@nivo/tooltip'
+import { memo } from 'react'
+import { useTheme } from '@nivo/core'
+
+interface BoxPlotSummaryTooltipProps {
+    label: string
+    formatted: BoxPlotSummaryFormatted
+    color?: string
+    enableChip?: boolean
+}
+
+export const BoxPlotSummaryTooltip = memo<BoxPlotSummaryTooltipProps>(
+    ({ label, formatted, enableChip = false, color }) => {
+        const theme = useTheme()
+
+        const quantiles = formatted.quantiles.map((q, i) => (
+            <div key={'quantile.' + i}>
+                {q * 100}%: <strong>{formatted.values[i]}</strong>
+            </div>
+        ))
+        return (
+            <div style={theme.tooltip.container}>
+                <div>
+                    <div style={theme.tooltip.basic}>
+                        {enableChip && <Chip color={color ?? ''} style={theme.tooltip.chip} />}
+                        {label}
+                    </div>
+                </div>
+                <br />
+                <div style={{ display: 'flex' }}>
+                    <span style={{ marginRight: '2rem' }}>
+                        <div>
+                            n: <strong>{formatted.n}</strong>
```

Comment:
Yes, I see the simplicity of the data+transformer approach, and the benefit of a component that is not specific to one language.

However, for the boxplot, displaying a single number in the tooltip would be quite disappointing. As a viewer, I would definitely like to see information on the number of points making up the boxplot (n), the numeric value for the median, some estimate of the box size. The current tooltip also has information about min/max values, which can be relevant sometimes but don't actually appear visually on the chart. So it is really valuable to display several numbers, and that means they must be annotated somehow.

One approach could be to provide different tooltips when the mouse hovers on the median line, the rectangle, the whiskers, etc. TBH, this would be a lot of work, and I think viewers would find it annoying not to see all the numbers all at once. Also, the tooltip would need to explain why it is displaying different values as the mouse hovers on different parts of the boxplot, so this might not solve the issue with language-specificity. 

I changed the implementation so that it is now possible to modify the static text via a translation dictionary (Is that what you meant by 'valuesLabels'). The component now expects to get this dictionary via the theme in `theme.translation`. It seems to work (there is now a story with tooltips in French). Is this acceptable? The translation could also go into a separate prop if you prefer.

Reply:
Another option/approach comes to mind now - we could keep the tooltip very simple (e.g. a formatted group name and a value for the median) and then provide a separate layer that can display additional information. That new layer would display a box in a static place on the chart, like a legend, but would update information on mouse hover/click. What do you think? An advantage would be that the tooltip would be very simple and not obstruct the chart underneath. But this would not replace the need for translations in the new info/legend box.

Reply:
I think it should go in a separate property.